Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+pipeline {
+    agent {
+        label 'kie-rhel7'
+    }
+    tools {
+        maven 'kie-maven-3.5.4'
+        jdk 'kie-jdk1.8'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 300, unit: 'MINUTES')
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                sh 'printenv'
+
+            }
+        }
+        stage('Build upstream projects') {
+            steps {
+                dir("droolsjbpm-build-bootstrap") {
+                    script {
+                        githubscm.checkoutIfExists('droolsjbpm-build-bootstrap', "$CHANGE_AUTHOR", "$CHANGE_BRANCH", 'kiegroup', "$CHANGE_TARGET")
+                        load("upstream.stages")
```

Comment:
Is it possible now or will it be possible in the future to reduce the repositories that will be built before this repository to just those that this project really depends on?

Reply:
yes, the next step is not make it smarter just building its dependencies (dynamically calculated)